R/FilteredData.R

```diff
@@ -586,6 +590,33 @@ FilteredData <- R6::R6Class( # nolint
     restore_state_from_bookmark = function(state, check_data_hash = TRUE) {
       stop("Pure virtual method")
     },
+    #' @description disable the filter panel
```

Comment:
```suggestion
    #' @description 
    #' Disable the filter panel by adding `disable` attribute to `filter_add_vars` and `filter_active_vars ` id in the UI element.
    #' In addition, it will store the existing filter states in a private field called `cached_states` 
    #' before remove all filter states from the object.
```